db/dataset.py

```diff
@@ -315,3 +315,54 @@ def _delete_snapshots_for_dataset(connection, dataset_id):
         DELETE FROM dataset_snapshot
               WHERE dataset_id = :dataset_id""")
     connection.execute(query, {"dataset_id": dataset_id})
+
+
+def add_recordings(dict, dataset_id):
+    """Adds new recordings to a class in a dataset"""
+
+    with db.engine.begin() as connection:
+        result = connection.execute(sqlalchemy.text("""SELECT id FROM dataset_class WHERE name = :name and dataset = :dataset_id"""),
```

Comment:
A reminder that we talked about formatting of SQL statements and making sure that lines are not too long